### ISSUE TITLE
Ally improvements 

### ISSUE THREAD

**Issue by erangatl** [wso2/product-apim#11457]
### Describe your problem(s)

![image](https://user-images.githubusercontent.com/20179540/126588203-4c683543-ed35-4610-b47f-a5b97e00973f.png)

ARIA hidden element must not contain focusable elements 1
ARIA input fields must have an accessible name 1
id attribute value must be unique 19
Certain ARIA roles must contain particular children 1
Heading levels should only increase by one 1
Document should have one main landmark 2
Ensures landmarks are unique 1
Page should contain a level-one heading 2
All page content should be contained by landmarks
add alt tag for all the images

### How will you implement it

---
